### PR TITLE
Add maintainer nomination template

### LINKED ISSUE #912: Minimum criteria for new maintainers?
Is there any minimum criteria/qualifications for a new maintainer to be added to this project?

For example:

- number of community calls attended
- involvement in mailing list
- involvement in WG efforts
- number of commits
- comment history

?

### PR DESCRIPTION
**Pull request by nishakm**
This is a PR template to propose adding new maintainers

Fixes #912

Signed-off-by: nisha <nisha@ctlfsh.tech>

### CODE REVIEW

**Comment by sajayantony** [comment]
+1

File: .github/PULL_REQUEST_TEMPLATE/maintainer_nomination.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Nomination for New Maintainers
```
Comment (jdolitsky) [inline .github/PULL_REQUEST_TEMPLATE/maintainer_nomination.md:1]: wondering if this should be amended to apply to one maintainer at a time?

Comment (nishakm) [reply .github/PULL_REQUEST_TEMPLATE/maintainer_nomination.md:1]: I'll update it

**Comment by sudo-bmitch** [comment]
I'd lean towards making the Justification section contain a comment describing possible ways to nominees could have activity, rather than fixed headings. It gives flexibility to participate in different ways. Some may attend meetings, respond on the mailing list, in slack, and I think all sorts of contributions should be valuable. The key part for me is whether they'll help respond to PRs since that's the primary responsibility for being a maintainer that I see.

I think we also want to look at what other templates would be useful, but that can be separate from this PR.

File: .github/PULL_REQUEST_TEMPLATE/maintainer_nomination.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Name of the new maintainer with GitHub username
+
+## Justification
+
+### Community Activity
+
+### Working Group Activity
+
+### OCI Contributions
```
Comment (jdolitsky) [inline .github/PULL_REQUEST_TEMPLATE/maintainer_nomination.md:17]: @nishakm - want to just remove these 3 headings for now so we can get this in?

**Comment by nishakm** [comment]
May 26 meeting: remove headings under justification and instead add comment on possible work a nominator can highlight

**Review by sudo-bmitch — APPROVED** [review]

**Review by jdolitsky — APPROVED** [review]
👍

**Comment by sudo-bmitch** [comment]
@opencontainers/image-spec-maintainers PTAL

**Review by mikebrow — APPROVED** [review]
LGTM

**Review by jonjohnsonjr — APPROVED** [review]

**Comment by jdolitsky** [comment]
Can we get a merge on this?

**Comment by Eldogor447-ui** [comment]
https://github.com/opencontainers/image-spec/pull/919#issue-1237430029

**Pull request merged**